.github/workflows/deploy_lambda_dev.yml

```
@@ -0,0 +1,24 @@
+name: Deploy Lambdas to Dev
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: Branch to deploy
+        required: true
+        default: master
+
+jobs:
+  setup-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.ref }}
+
+  build_deploy_lambda:
+    needs: [setup-job]
```

```suggestion
    needs: [setup_job]
```
Same comment as above.

---

Changed.